src/screencast/pipewire_screencast.c

```diff
@@ -230,13 +212,10 @@ void xdpw_pwr_export_buffer(struct xdpw_screencast_instance *cast) {
 		h->seq = cast->seq++;
 		h->dts_offset = 0;
 	}
-	if ((d[0].data) == NULL) {
-		logprint(TRACE, "pipewire: data pointer undefined");
-		goto queue;
-	}
 
-	writeFrameData(d[0].data, cast->screencopy_frame.data, cast->screencopy_frame.height,
-		cast->screencopy_frame.stride, cast->screencopy_frame.y_invert);
+	if (cast->screencopy_frame.y_invert) {
+		//TODO: Flip buffer or set stride negative
```

Comment:
Error out in the meantime?

Reply:
Added a naive implementation to reinvert the buffer.